packages/table/test/columnHeaderCellTests.tsx

```diff
@@ -106,6 +107,52 @@ describe("<ColumnHeaderCell>", () => {
             );
         });
 
+        describe("useInteractionBar", () => {
+            const NUM_COLUMNS = 5;
+            let consoleWarn: Sinon.SinonStub;
+
+            before(() => {
+                consoleWarn = sinon.stub(console, "warn");
+            });
+
+            afterEach(() => {
+                consoleWarn.reset();
+            });
+
+            after(() => {
+                consoleWarn.restore();
+            });
+
+            it("prints a deprecation warning if useInteractionBar=true", () => {
```

Comment:
I don't really like testing deprecations; these tests seem tautological and eat up build time. Would prefer to remove the first 3 tests here.

Reply:
Removed